src/main/scala/io/getclump/Utils.scala

```diff
@@ -0,0 +1,13 @@
+package io.getclump
+
+private[getclump] case class RunOnce[T, U](f: T => Clump[U]) extends (T => Clump[U]) {
```

Comment:
agree with attempting a better way to avoid var, even though Clump already has mutation elsewhere in ClumpFetcher